test/cluster/mzcompose.py

```diff
@@ -67,43 +67,9 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
     if shard_count:
         shard_count = int(shard_count)
 
-    for i, name in enumerate(
-        [
-            "test-smoke",
-            "test-github-12251",
-            "test-github-15531",
-            "test-github-15535",
-            "test-github-15799",
-            "test-github-15930",
-            "test-github-15496",
-            "test-github-17177",
-            "test-github-17510",
-            "test-github-17509",
-            "test-github-19610",
-            "test-single-time-monotonicity-enforcers",
-            "test-remote-storage",
-            "test-drop-default-cluster",
-            "test-upsert",
-            "test-resource-limits",
-            "test-invalid-compute-reuse",
-            "pg-snapshot-resumption",
-            "pg-snapshot-partial-failure",
-            "test-system-table-indexes",
-            "test-replica-targeted-subscribe-abort",
-            "test-replica-targeted-select-abort",
-            "test-compute-reconciliation-reuse",
-            "test-compute-reconciliation-no-errors",
-            "test-mz-subscriptions",
-            "test-mv-source-sink",
-            "test-query-without-default-cluster",
-            "test-clusterd-death-detection",
-            "test-replica-metrics",
-            "test-compute-controller-metrics",
-            "test-metrics-retention-across-restart",
-            "test-concurrent-connections",
-            "test-profile-fetch",
-        ]
-    ):
+    for i, name in enumerate(c.workflows):
+        if name in ("default", "test-incident-70"):
+            continue
```

Comment:
This ignores the execution of `test-incident-70`, right? Can we reference a GitHub issue here?

Reply:
Yes, because test-incident-70 requires more memory, thus it runs separately in nightly.